openlane/config/variable.py

```diff
@@ -447,7 +447,7 @@ def __process(
                     raw = raw.split(";")
                 else:
                     raw = raw.split()
-                if raw[-1] == "":
+                if len(raw) and raw[-1] == "":
```

Comment:
does this fix a bug?

Reply:
Yes, though this bug is in main and should be fixed there as well

Reply:
Then perhaps this should be a separate PR.